apps/site-nft/src/components/Claim/index.tsx

```diff
@@ -14,24 +14,27 @@ const messages = defineMessages({
   phTokenId: { defaultMessage: "ID of token purchased", description: "Placeholder for claim/tokenID input" },
   phCode: { defaultMessage: "Unlock code for the token purchased", description: "Placeholder for claim/code input" },
   claim: { defaultMessage: "Claim", description: "Claim button" },
+
+  success: { defaultMessage: "Success: <link>View the Transaction</link>", description: "Show on claim success" },
+  failed: { defaultMessage: "Claim Failed: Please contact support", description: "Show on claim failed" },
 })
 
 export const Claim = () => {
   const account = useActiveAccount()!;
   const [code, setCode] = useState('');
   const [tokenId, setTokenId] = useState('');
-  const [success, setSuccess] = useState(undefined as undefined | boolean);
+  const [success, setSuccess] = useState<string | boolean | undefined>();
   const intl = useIntl();
 
   const doClaimCode = async () => {
-    log.trace({address: account.address, tokenId}, "User {address} is claiming tokenId {tokenId}");
+    log.trace({ address: account.address, tokenId }, "User {address} is claiming tokenId {tokenId}");
     const api = GetNftApi();
     const r = await api.claimNft({
       tokenId: parseInt(tokenId),
       code,
       claimant: account.address,
     });
-    setSuccess(r.data);
```

Comment:
Typing can't handle unions in Swagger2.0 - this code will be removed as soon as we upgrade to Swagger 3